teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
* Kindly make the same fix here too as mentioned above since a `teacher` is inheriting from a `person ` that means a teacher is a type of person and so should inherit the base information needed to make a person ( name, age, parent_permission)

Reply:
Thanks for your suggestion. I have corrected it.

```
require './person'

class Teacher < Person
  def initialize(age, specialization, name = "Unknown")
    super(age, name)
    @specialization = specialization
  end

  def can_use_services?
    true
  end
end
```

Reply:
@Davidosky007 the below code is not producing any linter errors and also considering @parent_permission while creating `student` and `teacher` objects.

```
require './person'

class Student < Person
  def initialize(age, classroom, name = 'Unknown')
    super(age, name)
    @classroom = classroom
  end

  def play_hooky
    '¯(ツ)/¯'
  end
end
```

Terminal Output
```
[admin@ANTARCTICA oop-school-library]$ irb
irb(main):001:0> require './person.rb'
=> true
irb(main):002:0> require './student.rb'
=> true
irb(main):003:0> require './teacher.rb'
=> true
irb(main):004:0> student = Student.new(17, "class-A", "Hello")
=> #<Student:0x00005629b47694c0 @age=17, @classroom="class-A", @id=31, @name="Hello", @parent_permission=true>
irb(main):005:0> teacher = Teacher.new(55, 'science', "WORLD")
=> #<Teacher:0x00005629b46f05e8 @age=55, @id=35, @name="WORLD", @parent_permission=true, @specialization="science">
irb(main):006:0> 
```

![image](https://user-images.githubusercontent.com/40719899/198975675-7d4159cf-0426-4099-b434-cd63eeee414f.png)

I am raising a pull request with the above code.
Thank You